tests/integration/apigateway/__init__.py

```diff
@@ -0,0 +1,5 @@
+import os
+
+# parent directory of this file
+PARENT_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+OPENAPI_SPEC_PULUMI_JSON = os.path.join(PARENT_DIR, "files", "openapi.spec.pulumi.json")
```

Comment:
nit: we usually keep the `__init__.py` files clean of custom logic. Not critical, as this is just a test module, but something to consider for future iterations. 👍